2.6.md

```diff
@@ -2883,6 +2903,66 @@ This object should be included if the ad supported content is a Digital Out-Of-H
 |keywords    |string             |Comma separated list of keywords about the DOOH placement.                                                                                                                                                                               |
 |content     |object             |Details about the Content within the DOOH placement.                                                                                                                                                                     |
 |ext         |object             |Placeholder for exchange-specific extensions to OpenRTB.                                                                                                                                                                                 |
+### 3.2.33 - Object: Refresh <a name="objectrefresh"></a>
+
+The minimum exposure time, in seconds per view, that the (uninterrupted) ad creative will be rendered before refreshing to the next creative. If the field is absent, the exposure time is unknown. This field does not account for viewability or external factors such as a user leaving a page.
```

Comment:
this is very different than what was in the adx spec; needs rewording. 

![image (3)](https://user-images.githubusercontent.com/1683175/229956238-fd42c897-8844-4356-a130-85b7e559abb3.png)

Reply:
opened a pr to implement my suggestion https://github.com/InteractiveAdvertisingBureau/openrtb2.x/pull/54/files